src/Model/ValueObject/Calendar/OpeningHours/Time.php

```diff
@@ -4,6 +4,10 @@
 
 namespace CultuurNet\UDB3\Model\ValueObject\Calendar\OpeningHours;
 
+use CultuurNet\UDB3\DateTimeFactory;
+use DateTime;
+use DateTimeInterface;
+
 class Time
```

Comment:
I wonder if it would be a good idea to not put this in the OpeningHours namespace, but in the general ValueObject namespace, because this is such a broad and useful DTO.
I can imagine in the future using this class for other uses than the Calender.

Reply:
That is possible, but for now, the only supported format is `'H:i'` which is rather very specific in the context of opening hours and not in the sense of a time of the day.